src/raw.js

```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = class Raw {
+  constructor(value) {
+    this.value = value;
+    // consumed in expr_formatter.js
+    this.type = 'raw';
+  }
+};
```

Comment:
bit worried that the original raw data structure might be supplied by user input directly, such as `where({ name: { __raw: true, value: "'' OR 1" })`. Hence changed it to a specific js class that needs instantiation.